simulator/src/data/backupCircuit.js

```diff
@@ -32,6 +32,9 @@ export function backUp(scope = globalScope) {
     // Storing all nodes
     data.allNodes = scope.allNodes.map(extract);
 
+    // Storing test attached to scope
+    data.test = scope.test;
```

Comment:
```suggestion
    data.testBenchData = scope.testBenchData;
```
Rename variable to be more descriptive